docs/chapters/baseplatform/replacing-packagegroups.rst

```diff
@@ -0,0 +1,57 @@
+:orphan:
+
+.. _replacing-packagegroups:
+
+Replacing package groups by hand-picked components
+==================================================
+
+By default PELUX include components from QT Auto using the package groups feature.
```

Comment:
s/QT/Qt the name of the product is `Qt`.

Reply:
Should we prefer "QtAuto" instead of "Qt Auto"?

Reply:
Qt Auto should be fine :)